config/http.js

```diff
@@ -3,85 +3,53 @@
-   *                                                                          *
-   * The body parser that will handle incoming multipart HTTP requests. By    *
-   * default as of v0.10, Sails uses                                          *
-   * [skipper](http://github.com/balderdashy/skipper). See                    *
-   * http://www.senchalabs.org/connect/multipart.html for other options.      *
-   *                                                                          *
-   ***************************************************************************/
-
-  // bodyParser: require('skipper')
-
-  // },
-
-  /***************************************************************************
-   *                                                                          *
-   * The number of seconds to cache flat files on disk being served by        *
-   * Express static middleware (by default, these files are in `.tmp/public`) *
-   *                                                                          *
-   * The HTTP static cache is only active in a 'production' environment,      *
-   * since that's the only time Express will cache flat-files.                *
-   *                                                                          *
-   ***************************************************************************/
-
-  // cache: 31557600000
+  middleware: {
+    /***************************************************************************
+     *                                                                          *
+     * The order in which middleware should be run for HTTP requests.           *
+     * (This Sails app's routes are handled by the "router" middleware below.)  *
+     *                                                                          *
+     ***************************************************************************/
+    // order: [
+    //   'cookieParser',
+    //   'session',
+    //   'bodyParser',
+    //   'compress',
+    //   'poweredBy',
+    //   'router',
+    //   'www',
+    //   'favicon',
+    // ],
+    /***************************************************************************
+     *                                                                          *
+     * The body parser that will handle incoming multipart HTTP requests.       *
+     *                                                                          *
+     * https://sailsjs.com/config/http#?customizing-the-body-parser             *
+     *                                                                          *
+     ***************************************************************************/
+    // bodyParser: (function _configureBodyParser(){
+    //   var skipper = require('skipper');
+    //   var middlewareFn = skipper({ strict: true });
+    //   return middlewareFn;
+    // })(),
+  },
+
+  // https://sailsjs.com/documentation/reference/configuration/sails-config-http#?properties
+  trustProxy: true,
```

Comment:
Made sure to keep your `trustProxy` key and verified it still exists since you need it for Heroku.